### PR TITLE
Small code improvements

### PR DESCRIPTION
**Pull request by kukulich**

### CODE REVIEW

File: src/SourceLocator/SourceStubber/ReflectionSourceStubber.php [hunk 1]
```diff
@@ -76,21 +76,20 @@ public function __construct()
...
             || interface_exists($className, false)
             || trait_exists($className, false)
-            || $isEnum
             )
         ) {
             return null;
         }
 
+        /** phpcs:disable SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName */
+        $isEnum = function_exists('enum_exists') && \enum_exists($className, false);
```
Comment (Ocramius) [inline src/SourceLocator/SourceStubber/ReflectionSourceStubber.php:90]: do enums pass `class_exists()` checks?

File: src/SourceLocator/SourceStubber/ReflectionSourceStubber.php [hunk 2]
```diff
@@ -76,21 +76,20 @@ public function __construct()
...
             class_exists($className, false)
             || interface_exists($className, false)
             || trait_exists($className, false)
-            || $isEnum
             )
         ) {
             return null;
         }
 
+        /** phpcs:disable SlevomatCodingStandard.Namespaces.ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName */
```
Comment (Ocramius) [inline src/SourceLocator/SourceStubber/ReflectionSourceStubber.php:89]: I suppose importing the function won't work due to PHP 8.0 and static analysis?

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: src/SourceLocator/SourceStubber/ReflectionSourceStubber.php (hunk 1 above)
Comment (kukulich) [reply src/SourceLocator/SourceStubber/ReflectionSourceStubber.php:90]: Yes. I just figured it out. 🤷‍♂️ 

Here I need preserve the check to choose `ReflectionEnum` or `ReflectionClass`.

File: src/SourceLocator/SourceStubber/ReflectionSourceStubber.php (hunk 2 above)
Comment (kukulich) [reply src/SourceLocator/SourceStubber/ReflectionSourceStubber.php:89]: It works but PHPStan does not like `use function` for missing functions so this is hack for PHPStan. Consulted with @ondrejmirtes

**Review by Ocramius — APPROVED** [review]
LGTM :ship:

**Comment by Ocramius** [comment]
Thanks @kukulich!

**Pull request merged**